postprocessing_variant_calls/maf/helper.py

```diff
@@ -446,6 +445,68 @@ def filter(self, filter):
             raise typer.Abort()
         return self.data_frame
 
+    # def tag_exonic_variant(self):
+    #     self.data_frame["is_exonic_variant"] = self.data_frame["Variant_Classification"].apply(lambda x: 'yes' if x in self.keep_exonic else 'no')
+    #     return self.data_frame
+
+    # def tag_tert_variant(self):
+    #     self.data_frame['is_tert_variant'] = (self.data_frame['Variant_Classification'] == "5'Flank") & (self.data_frame['Hugo_Symbol']== 'TERT')
+    #     custom_values = {False: 'no', True: 'yes'}
+    #     self.data_frame['is_tert_variant'] = self.data_frame['is_tert_variant'].replace(custom_values)
+    #     return self.data_frame
+
+    def tag_variant_by_intervals(self, intervals_file):
+        rules_df = pd.read_csv(intervals_file)
+
+        rules_df.fillna("none", inplace=True)
+
+        for index, row in rules_df.iterrows():
```

Comment:
We'll want to define this rules file a little more strictly. I think we had some good discussion. Let's try a JSON file and using a specific function / class to read in said JSON file.